core-tests/shared/src/test/scala/zio/ChunkSpec.scala

```diff
@@ -165,28 +165,28 @@ object ChunkSpec extends ZIOBaseSpec {
     testM("specialized accessors") {
```

Comment:
Good catch! Let's just make this a suite and each of the checks its own test within the suite.